### PR TITLE
Registration Kiel to Up-for-Grabs

### PR DESCRIPTION
**Pull request by ibrahimyilmaz**
Hi there,
I would like to register my open source project to Up-For-Grabs.
This PR is basically about that.

Sincerely
Ibra

### CODE REVIEW

File: _data/projects/kiel.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+site: https://ibrahimyilmaz.github.io/kiel/
+tags:
+  - kotlin
+  - recyclerview
+  - android
+  - androiddev
+  - viewholder
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/ibrahimyilmaz/kiel/labels/up%20for%20grabs
```
Comment (ritwik12) [inline _data/projects/kiel.yml]: @ibrahimyilmaz I think you mistyped the Label URL
```suggestion
  link: https://github.com/ibrahimyilmaz/kiel/labels/up-for-grabs
```

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/kiel.yml (hunk 1 above)
Comment (ibrahimyilmaz) [reply _data/projects/kiel.yml]: It is not miss typing. It is written so in the up for grabs official documentation!
Up-for-grabs or up for grabs, the difference is this. So your thinking is false but anyway i did your suggestion.

Comment (ritwik12) [reply _data/projects/kiel.yml]:
```
upforgrabs:
  name: up for grabs
  link: https://github.com/username/project/labels/up%20for%20grabs
```
You meant this in https://github.com/up-for-grabs/up-for-grabs.net/blob/gh-pages/docs/list-a-project.md

here you did not see that it was `name: up for grabs` where as you used ` name: up-for-grabs`

Comment (ritwik12) [reply _data/projects/kiel.yml]: @ibrahimyilmaz

**Review by ritwik12 — APPROVED** [review]

File: _data/projects/kiel.yml (hunk 1 above)
Comment (ibrahimyilmaz) [reply _data/projects/kiel.yml]: So what I should do now?

**Pull request merged**